core/app/services/spree/products/translations/generate_automated_translations.rb

```diff
@@ -0,0 +1,69 @@
+module Spree
+  module Products
+    module Translations
+      class GenerateAutomatedTranslations
+        prepend Spree::ServiceModule::Base
+
+        def initialize(tp: self.class.injected_automated_translations_provider)
```

Comment:
I would prefer to name the var `provider` instead of tp, I'm generally not a fan of these single-character abbreviations. But this is personal preference.

Reply:
I wonder how I ended up calling it `tp` in the first place 🙄